giskard/ml_worker/websocket/listener.py

```diff
@@ -221,10 +223,15 @@ async def wrapped_callback(req: ActionPayload, client_params: Dict, worker_info,
     return websocket_actor_callback
 
 
+@websocket_actor(MLWorkerAction.abort, execute_in_pool=False)
+def on_abort(ml_worker: MLWorkerInfo, params: AbortParams, *args, **kwargs):
+    POOL.cancel(params.job_id)
```

Comment:
This should be wrapped, to hide the pool (same as for run_in_pool), for a matter of uniformity

Reply:
so you should just call a `cancel_in_pool(job_id)`

Reply:
not sure I get it. Does the `cancel_in_pool` exist already?

Reply:
Why do you want to hide the POOL object inside utils?

Reply:
done